pkg/oc/cli/cmd/export.go

```diff
@@ -133,12 +142,44 @@ func RunExport(f *clientcmd.Factory, exporter Exporter, in io.Reader, out io.Wri
 		newInfos := []*resource.Info{}
 		errs := []error{}
 		for _, info := range infos {
+			converted := false
+
+			// convert unstructured object to runtime.Object
```

Comment:
I'm ok with the approach, but only if we deprecate `oc export` and stop adding any new features to it.  

@fabianofranz it's a pretty ugly hammer, but it behaves slightly better on unknown types.